audio-worklet/basic/audio-worklet-node-options/basic-processor.js

```diff
@@ -0,0 +1,66 @@
+
+// https://en.wikipedia.org/wiki/Sawtooth_wave
+const sawtooth = (f, t) => {
+  return 2 * (t * f - Math.floor(t * f + 0.5));
+}
+
+// https://en.wikipedia.org/wiki/Sine_wave
+const sine = (f, t) => {
+  return Math.sin(2 * Math.PI * f * t);
+}
+
+// https://en.wikipedia.org/wiki/Square_wave
+const square = (f, t) => {
+  return 2 * (2 * Math.floor(f * t) - Math.floor(2 * f * t)) + 1
+}
+
+// https://en.wikipedia.org/wiki/Triangle_wave
+const triangle = (f, t) => {
+  return 2 * Math.abs(2 * (f * t - Math.floor(f * t + 0.5))) - 1
+}
+
+const noise = () => {
+  return 2 * (Math.random() - 0.5);
+}
+
+class BasicProcessor extends AudioWorkletProcessor {
```

Comment:
Add an overview comment for the class definition:
https://google.github.io/styleguide/jsguide.html#jsdoc-class-comments